<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon_rec.qrc">
    <normaloff>:/icons/icons/icon.svg</normaloff>:/icons/icons/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_contacts" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>205</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>178</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>205</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget_contacts"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_head">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_rec.qrc">:/icons/icons/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_userName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>2</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_fontSize">
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image:url(&quot;:/A26_2.png&quot;);
}
QLabel:hover{
  border:2px solid rgb(203, 203, 203);
	background-color: rgb(191, 197, 204);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_rec.qrc">:/icons/icons/font_size.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_emoji">
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image:url(&quot;:/A26_2.png&quot;);
}
QLabel:hover{
  border:2px solid rgb(203, 203, 203);
	background-color: rgb(191, 197, 204);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_rec.qrc">:/icons/icons/emoji.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sendimage">
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image:url(&quot;:/A26_2.png&quot;);
}
QLabel:hover{
  border:2px solid rgb(203, 203, 203);
	background-color: rgb(191, 197, 204);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_rec.qrc">:/icons/icons/image.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sendFile">
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image:url(&quot;:/A26_2.png&quot;);
}
QLabel:hover{
  border:2px solid rgb(203, 203, 203);
	background-color: rgb(191, 197, 204);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_rec.qrc">:/icons/icons/sendfile.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>422</y>
      <width>741</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(130, 130, 130);</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
       <property name="autoRepeatDelay">
        <number>301</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(130, 130, 130);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon_rec.qrc"/>
 </resources>
 <connections/>
</ui>
